aemedge/blocks/hero/hero.js

```diff
@@ -118,20 +165,29 @@ export default async function decorate(block) {
     }
   });
 
+  // Add Primary tag
+  const tagContainer = div({ class: 'media-blend__tags' });
+  const firstTagText = getMetadata('article:tag').split(',')[0].trim();
```

Comment:
Can you already read this from `topics` tag, as `article:tag` will be replaced

Reply:
I might be mistaken, but seems like `topic` rather than `topics` is the new name for the property. Have updated to use `topic` instead.